### DISCUSSION TITLE
Abort $fetch request

### DISCUSSION THREAD

**Discussion by bloor** [nuxt/nuxt#17618] [Ideas]
I noticed there is no such option available right now, both in nuxt and in ohmyfetch git.
What would be nice to have:

1) abort a fetch request on demand
2) abort a previous unfinished fetch request if duplicate new request is made; could be manually implemented using 1, but an out of the box solution is always great to have.
3) auto-abort option for all request made from a component, onUnmounted();

**Comment by asonnleitner** [comment]
Would be nice to have this implemented in the useFetch instead implementing own fetch handler in useAsyncData. e.g. https://use-http.com/#/?id=abort
